### ISSUE TITLE
Pasting From History Doesn’t Work After a bit

### ISSUE THREAD

**Issue by shadoez** [Traurige/Kayoko#4]
**Describe the bug:**
Tapping something I copied from my history will not paste on first attempt, but works on the second. This happens even with Automatically Paste enabled.

**Steps to reproduce the behavior:**
Leave the history list alone for around 10 minutes, try to paste again & nothing will happen.

**System information:**
- iOS version: 15.0.2
- Jailbreak: XinaA15

**Comment by Traurige** [comment]
Yeah, I can see what you mean. Not sure what the issue is yet, but I'll try to figure it out :)
